firmware/application/hw/encoder.hpp

```diff
@@ -32,6 +32,7 @@ class Encoder {
 
    private:
     uint_fast8_t state{0};
+    uint_fast16_t store{0};
```

Comment:
I would still change this to a uint_fast8_t, even though it seems it's apparently holding more than 8 bits.